Loop/Managers/TemporaryPresetsManager.swift

```diff
@@ -165,11 +188,11 @@ class TemporaryPresetsManager {
     }
 
     public func nonPreMealOverrideEnabled(at date: Date = Date()) -> Bool {
-        return scheduleOverride?.isActive(at: date) == true
+        return scheduleOverrideEnabled(at: date) == true && scheduleOverride?.context != .preMeal
```

Comment:
Generally prefer the former "is active at", rather than "enabled at", as the latter could be read as the specific point in time that the preset was started/enabled.

Reply:
I believe this was the naming that was there originally. Let me see what'd be involved in updating!

Reply:
Okay, updated.